apps/mobile/src/components/search/TokenLaunchCard/index.tsx

```diff
@@ -64,10 +63,53 @@ export const TokenLaunchCard: React.FC<LaunchCoinProps> = ({
   return (
     <View style={[styles.container, isDesktop && styles.containerDesktop]}>
       <View style={styles.header}>
+        {token?.created_at ? (
+          <Text style={styles.creationTime}>
+            {getElapsedTimeStringFull(new Date(token.created_at).getTime())}
+          </Text>
+        ) : null}
         <Text style={styles.tokenName}>{token?.name || 'Unnamed Token'}</Text>
         {token?.symbol ? <Text style={styles.tokenSymbol}>{token.symbol}</Text> : null}
         <Text style={styles.price}>${Number(token?.price || 0).toFixed(4)}</Text>
       </View>
+      <View style={styles.divider} />
+      {token?.threshold_liquidity && (
+        <View>
+          <View style={styles.detailRow}>
+            <Text style={styles.label}>Raised Progress</Text>
+            <Text style={styles.value}>
+              {Number(token?.liquidity_raised || 0).toLocaleString()} /{' '}
+              {Number(token?.threshold_liquidity).toLocaleString()}
+            </Text>
+          </View>
+          <View style={styles.progressBarContainer}>
+            <View
+              style={[
+                Number(token.liquidity_raised) <= Number(token.threshold_liquidity) - 0.2
```

Comment:
Same comment as above, used a slippage tolerance of 2 percent and not a brut one 2 percent, so if we change the threshold it's still ok